### PR TITLE
Implement Explorer workflow for STAGE, PROD

### PR DESCRIPTION
**Pull request by DmitriyMolch**
https://cerenetwork.atlassian.net/browse/CBI-2322

### CODE REVIEW

File: .github/workflows/explorer-prod.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+name: Enable Explorer maintenance on PRD
```
Comment (AndreiNavoichyk) [inline .github/workflows/explorer-prod.yaml]: Rename `PRD` to `PROD`

File: .github/workflows/explorer-stage.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+name: Enable Explorer maintenance on DEV
```
Comment (AndreiNavoichyk) [inline .github/workflows/explorer-stage.yaml]: Here should be `STAGE`

File: .github/workflows/explorer-prod.yaml (hunk 1 above)
Comment (DmitriyMolch) [reply .github/workflows/explorer-prod.yaml]: fixed

File: .github/workflows/explorer-stage.yaml (hunk 2 above)
Comment (DmitriyMolch) [reply .github/workflows/explorer-stage.yaml]: fixed

**Pull request merged**